srtcore/api.cpp

```diff
@@ -926,7 +1080,7 @@ int CUDTUnited::close(const SRTSOCKET u)
        CTimer::triggerEvent();
    }
 
-   HLOGC(mglog.Debug, log << "%" << u << ": GLOBAL: CLOSING DONE");
+   HLOGC(mglog.Debug, log << "@" << u << ": GLOBAL: CLOSING DONE");
```

Comment:
You'll find more replacement from `%` into `@` in the logs. This was a mistake earlier to use the `%` as a number prefix here. The `%` is meant to be a prefix for a sequence number, while `@` is to be a prefix for a socket id.